.eslintrc.js

```diff
@@ -59,7 +59,7 @@ module.exports = {
     'no-empty': 'off',
     'no-extra-boolean-cast': 'off',
     'no-fallthrough': 'off',
-    'no-nested-ternary': 'error',
+    'no-nested-ternary': 'off',
```

Comment:
This is a rather big change. Is there a reason for this other than to accommodate just this PR?